tests/ReflectionTarget.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+namespace Brick\Reflection\Tests;
+
+/**
+ * The Reflection Target class.
+ */
+class ReflectionTarget
+{
+    /**
+     * @param string $foo
+     */
+    private $foo;
+
+    /**
+     * @var string $bar
+     */
+    private $bar;
+
+    /**
+     * @var \\Exception $barWithType
+     */
+    private $barWithType;
+
+    public function __construct()
+    {
+        $this->foo = 'foo';
+        $this->bar = 'bar';
+    }
+
+    /**
+     * @param  string $str
+     * @return string $str
+     */
+    private function privateFunc(string $str)
+    {
+        return $str;
+    }
+
+    /**
+     * @return void
+     */
+    public static function publicStaticMethod()
+    {
+        return 'publicStaticMethod';
```

Comment:
You don't need to return anything here. You've declared it `@return void`, let's remove the body.